interfaces/acados_template/acados_template/c_templates/main.in.c

```diff
@@ -94,7 +76,6 @@ external_function_param_casadi * sim_impl_dae_fun_jac_x_xdot_z;
 external_function_param_casadi * sim_impl_dae_jac_x_xdot_u_z;
```

Comment:
In tera the `acados_sim_solver` is not used in the `main.in.c` file.
I think it would be great to track the differences in the tera, jinja2 template.
I guess the plan is still to use only use `tera` eventually.

Beside the fact that acados_sim should be used in the same way in the main templates, do you remember something else?
It would be good to track these todos in the `ROADMAP.md`.

Reply:
Yeah, I want to get rid of Jinja as soon as IFAC deadline is over :P